docs/src/man/subsets.md

```diff
@@ -1,109 +0,0 @@
-# Subsets
```

Comment:
This file was missing from the ToC. It's redundant with the new contents, except for `filter` and `filter!`, which don't really like advertising given that they are quite slow currently.